lib/matplotlib/colorbar.py

```diff
@@ -548,7 +548,6 @@ def _config_axis(self):
                       ticks_position=self.ticklocation)
         short_axis.set_ticks([])
         short_axis.set_ticks([], minor=True)
-        self._set_label()
```

Comment:
Does it still make sense to set `self.stale = True` here though?

Reply:
indeed.